app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find_by_id(params[:id])
+  end
```

Comment:
- [ ] Nice work creating the `users_controller.rb` with the `#index` and `#show` actions :clap: However, it would be nice to add a corresponding view files for each of the `users_controller` actions to follow the project requirement.
**Example:** 
```ruby
    # app/views/users/index.html.erb
    # app/views/users/show.html.erb
```
![Screenshot from 2023-12-19 19-19-33](https://github.com/BHObserver/blog-app/assets/91142432/da0df800-b6a6-4df6-bf5f-6e03349945e8)